### DISCUSSION TITLE
Can we narrow search results by upload date?

### DISCUSSION THREAD

**Discussion by Flopblopper** [FreeTubeApp/FreeTube#3972] [Q&A] [answered]
Hello,

I'm trying to figure out if it's possible to narrow search results by when they were uploaded, similar to YouTube's website. I'm looking at the search page and I don't see any obvious way to do it, but I'm hoping that I'm missing something.

Thanks

**Comment by absidue** [comment, marked as answer]
You can click on funnel next to the search bar to open the search filters menu. You need to set them before searching or trigger the search again after you've set them, to avoid sending extra requests to YouTube they aren't applied live.

**Reply by Flopblopper** [reply]
Thanks. I'm blind, and my screen reading software was having trouble with that button, so I didn't notice it was there.

**Reply by absidue** [reply]
I'm sorry to hear you are having issues using FreeTube with a screen reader. Looks like that button doesn't have a text label, just an icon, which is probably why your screen reader was struggling. While we've made some big improvements to accessibility recently, I'm sure there is still more to do, so all I can say is that things should get even better as time goes on.
